extension/src/project/projectCommands.ts

```diff
@@ -99,6 +100,45 @@ export function registerProjectCommands(context: vscode.ExtensionContext) {
             }
         }));
 
+        context.subscriptions.push(vscode.commands.registerCommand('autolisp.addWorkspaceFile2Project', async (clickedFile: vscode.Uri, selectedFiles: vscode.Uri[]) => {
+            if (!selectedFiles || selectedFiles.length === 0 || getWarnIsSearching()){
+                return;
+            }
+            let selectedDirs = selectedFiles.filter(f => fs.statSync(f.fsPath, { bigint: false}).isDirectory());
+            selectedFiles = selectedFiles.filter(f => fs.statSync(f.fsPath, { bigint: false}).isFile());
```

Comment:
Your codes cthere support to select a dir. But in the project node it doesn't support to select dir. I think to keep consisstene in hte project node context menu it is best to have this.

Reply:
Okay, but I am not very good at designing icons. So, which SVG's would you like associated with this new interface element?

Reply:
I only mean to refine the functionality of the menu item to support selecting dir, see pic:
![addfilefromworkspace-menu](https://user-images.githubusercontent.com/68938334/99902006-8b993e00-2c6f-11eb-95e7-d88cb38d2d75.gif)